doc/api/api_changes.rst

```diff
@@ -21,11 +21,11 @@ added to Matplotlib, see :ref:`whats-new`.
    .. note::
 
       The list below is a table of contents of individual files from the
-      'next_api_changes' folder.
+      'recent_api_changes_3.3' folder.
 
       When a release is made
 
-       - The files in 'next_api_changes/' should be moved to a new file in
+       - The files in 'recent_api_changes_3.3/' should be moved to a new file in
```

Comment:
Since we don't use a generic name anymore, there is no need to move the files.

Reply:
So, I have to delete this. Deleting it.